### ISSUE TITLE
Your demo link is down :)

### ISSUE THREAD

**Issue by chasemp** [trebuchet-deploy/ricochet#6]
Wasn't sure if you knew.

**Comment by ryan-lane** [comment]
Yeah. I really need to get that back up! I have some basic work at making a cluster in AWS. I need to get some time to finish that up.
